#!/usr/bin/env python
PACKAGE = "line_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("show_images", bool_t, 0, "Show some images when it is true", False)
gen.add("template_format", str_t, 0, "N,M,L: NxM zeros matrix and N pixel width in the middle are 255", "10,3,2")
gen.add("thresh_before_temp_match", int_t, 0, "Threshold of similarity before template matching", 185, 0, 255)
gen.add("thresh_result_temp_match", double_t, 0, "Threshold of similarity after template matching", .62, 0, 1)

exit(gen.generate(PACKAGE, "line_detection", "reconfig"))
